tasks/vendoring/__init__.py

```diff
@@ -83,8 +83,8 @@ def rewrite_file_imports(item, vendored_libs):
             text,
         )
         text = re.sub(
```

Comment:
Could we add a comment here - something along the lines of "we require a dot or whitespace after the library name, so we don't get a false positive when the library name is a prefix of another (see github issue #xxxx)"?

Reply:
Well either you comment every single part of the regex or you don't, I don't think this one is particularly non-obvious or extra tricky to deserve a special treatment. You do? Whoever is interested can see the explanation in the commit.

Reply:
It's basically an equivalent to `(\n\s*)` in the regex above.

Reply:
Ah, yes - you're right the parallel with the other regex is sufficient.